plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/AbstractContentSchemaProvider.java

```diff
@@ -21,13 +21,13 @@ public abstract class AbstractContentSchemaProvider
         implements ContentSchemaProvider
 {
     @Override
-    public final Optional<String> readKeyContentSchema(KafkaTableHandle tableHandle)
+    public final Optional<String> getKey(KafkaTableHandle tableHandle)
     {
         return readSchema(tableHandle.getKeyDataSchemaLocation(), tableHandle.getKeySubject());
     }
 
     @Override
-    public final Optional<String> readValueContentSchema(KafkaTableHandle tableHandle)
+    public final Optional<String> getValue(KafkaTableHandle tableHandle)
```

Comment:
getValue -> getMessage?

Reply:
sure, will do! Just for context: schema registry uses the term "value" for kafka "message" subject, not sure why since they're both confluent products. I'll change it shortly.